utils/build-script-impl

```diff
@@ -159,9 +159,9 @@ KNOWN_SETTINGS=(
     enable-llvm-assertions      "1"              "set to enable llvm assertions"
     build-llvm                  "1"              "set to 1 to build LLVM and Clang"
     build-swift-tools           "1"              "set to 1 to build Swift host tools"
-    build-swift-stdlib          "1"              "set to 1 to build the Swift standard library"
     build-swift-stdlib-unittest-extra "0"        "set to 1 to build optional StdlibUnittest components"
     build-swift-sdk-overlay     "1"              "set to 1 to build the Swift SDK overlay"
```

Comment:
Wouldn't the equivalent after these changes be `--build-swift-dynamic-stdlib=0 --build-swift-static-stdlib=0 --build-swift-sdk-overlay=0`? Or are you saying that you'd prefer that the `--build-swift-stdlib` parameter remain? My bad, but I'm having trouble grasping what you mean.